src/api/public/apidocs-new/paths/build_project_repository_architecture_package_file.yaml

```
@@ -0,0 +1,39 @@
+get:
+  summary: This endpoint returns a specific artifact file contents
```

```suggestion
  summary: Return a specific artifact file contents
```

Nitpick: This is something similar to what it is done with the commit message rule: "Use the imperative mood in the subject line".

This also applies for the rest of the summaries of the endpoints.

---

Fixed